### PR TITLE
Files for pushing the image to the hub.

### PR DESCRIPTION
**Pull request by Tor101**
I'm not sure if I've done it right. But I tried my best. I was trying to put it up in the hub. But I don't have privileges etc.

### CODE REVIEW

File: worker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+CMD ["npm", "run", "dev"]
```
Comment (chitalian) [inline worker/Dockerfile:12]: is there any reason this isn't using wrangler?

**Review by chitalian — COMMENTED** [review]
This is awesome thanks for this!

File: .github/workflows/docker-image.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: ghcr.io/tor101/helicone/helicone:latest
```
Comment (chitalian) [inline .github/workflows/docker-image.yml:25]: This is awesome

**Comment by chitalian** [comment]
This is great - the team and I will test these changes out. 

Related PR: https://github.com/Helicone/helicone/pull/244

**Comment by Tor101** [comment]
Closing this cause the pull was a success. Great work guys!

**Pull request closed without merging**